icechunk-python/tests/test_concurrency.py

```diff
@@ -70,6 +70,9 @@ async def test_concurrency() -> None:
                     write_to_store(array, x, y, barrier), name=f"write {x},{y}"
                 )
 
+    all_coords = {coords async for coords in session.chunk_coordinates("/array")}
+    assert all_coords == {(x, y) for x in range(N) for y in range(N)}
```

Comment:
could be nice to add a test where `y` goes to `N-1`

Reply:
done